server/utils/utils.go

```diff
@@ -0,0 +1,14 @@
+package utils
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+)
+
+func HashUUID(UUID string) string {
+	hash := sha256.New()
+	hash.Write([]byte(UUID))
+	md := hash.Sum(nil)
+	mdStr := hex.EncodeToString(md)
+	return mdStr
```

Comment:
```go
func sha256(str string) string {
	hash := sha256.New()
	hash.Write([]byte(str))
	md := hash.Sum(nil)
	return fmt.Sprintf("%s", hex.EncodeToString(md))
}

func GenerateVethName(uuid) string {
  str := sha256(uuid)
  return fmt.Sprintf("veth%s", str[0:8])
}
```

Reply:
It should be `fmt.Sprintf("%x", hex.EncodeToString(md))` I think, not `%s`.

Reply:
BTW, refactor is needed because the `sha256` function is defined and used in more than one packages.
We can move it to `github.com/linkernetworks/utils` and in other packages, just import it.